### PR TITLE
Update Docs to mention Gatsby plugin rename

### PR DESCRIPTION
**Pull request by umxr**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes
      /start features)

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

The documentation regarding the gatsby plugin doesn't mention the rename of the plugin

## What is the new behavior?

The documentation now mentions the rename of the gatsby plugin first so users don't get confused. ive updated the with-gatsby.mdx file. if there are more files that need to be updated then please let me know :) 

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: website/pages/docs/migration.mdx [hunk 1]
```diff
@@ -349,6 +349,12 @@ export default class Document extends NextDocument {
 
 Here's how to add it for Gatsby:
 
+## Changes
+
+- The Gatsby plugin has been renamed from `gatsby-plugin-chakra-ui` to
+  `@chakra-ui/gatsby-plugin`. Please make sure to have updated this when
+  installing ChakraUI in your next Gatsby project.
```
Comment (jmiazga) [inline website/pages/docs/migration.mdx:356]: Can you add a space so it's `Chakra UI`?

File: website/pages/guides/integrations/with-gatsby.mdx [hunk 2]
```diff
@@ -5,6 +5,14 @@ tags: ["gatsby", "websites", "ssr"]
 author: KenzoBenzo
 ---
 
+# Migration Notes
+
+## Changes
+
+- The Gatsby plugin has been renamed from `gatsby-plugin-chakra-ui` to
+  `@chakra-ui/gatsby-plugin`. Please make sure to have updated this when
+  installing ChakraUI in your next Gatsby project.
```
Comment (jmiazga) [inline website/pages/guides/integrations/with-gatsby.mdx:14]: Same as aboce, can you add a space so it's `Chakra UI`?

**Review by jmiazga — CHANGES_REQUESTED** [review]

File: website/pages/docs/migration.mdx (hunk 1 above)
Comment (umxr) [reply website/pages/docs/migration.mdx:356]: resolved in latest push :)

File: website/pages/guides/integrations/with-gatsby.mdx (hunk 2 above)
Comment (umxr) [reply website/pages/guides/integrations/with-gatsby.mdx:14]: resolved in latest push :)

File: website/pages/docs/migration.mdx (hunk 1 above)
Comment (jmiazga) [reply website/pages/docs/migration.mdx:356]: Hmm i don't see a change

Comment (umxr) [reply website/pages/docs/migration.mdx]: sorry just came through now, the pushed failed due to my internet haha

**Comment by jmiazga** [comment]
Thanks for the contribution! @allcontributors please add @umxr for docs

**Review by jmiazga — APPROVED** [review]

**Pull request merged**